File: ejer-6.py
#verificar si una palabra es palindrome
cad = input("Cad: ")
cad = cad.lower()
cad = cad.replace('?','')
cad = cad.replace(',','')
cad = cad.replace('.','')
cad = cad.replace(' ','')
print(cad == cad[::-1])

File: ejer-6.js
/*
Verificar si una palabra es palindrome
*/
const pali = (cadena) => {
   cadena = cadena.toLowerCase()
   cadena = cadena.replace(/\.+|\,+|\?+|\!+|\s+/g,'')
   let arr = cadena.split('')
   let rever = arr.reverse().join('')
   return cadena == rever
}
// pali('betto')
console.log(pali('oruro'))